app/src/main/java/com/keylesspalace/tusky/db/AppDatabase.java

```diff
@@ -846,7 +846,7 @@ FOREIGN KEY(`reportId`, `tuskyAccountId`) REFERENCES `NotificationReportEntity`(
     public static final Migration MIGRATION_60_62 = new Migration(60, 62) {
         @Override
         public void migrate(@NonNull SupportSQLiteDatabase database) {
-            database.execSQL("ALTER TABLE `AccountEntity` ADD COLUMN `defaultReplyPrivacy` INTEGER NOT NULL DEFAULT 2");
+            database.execSQL("ALTER TABLE `AccountEntity` ADD COLUMN `defaultReplyPrivacy` INTEGER NOT NULL DEFAULT 0");
```

Comment:
This only affects users that upgrade after this change, others will be stuck with "unlisted" but there is nothing we can do about this.